config.json

```diff
@@ -22,7 +22,16 @@
                     "name": "SDK Configuration"
                 },
                 {
-                    "name": "Integration Basics"
+                    "name": "Java Suite Integration Basics",
+                    "platforms": ["Android", "JavaSE", "JavaEE"]
+                },
+                {
+                    "name": "iOS Integration Basics",
```

Comment:
Is is possible to remove "iOS", "JavaScript Suite", and "Java Suite" from the name? It seems redundant.

Reply:
To have separate files it is not.